### DISCUSSION TITLE
Regarding the custom Effect like Snapchat have.

### DISCUSSION THREAD

**Discussion by shubham-ghanghotia** [shogo4405/HaishinKit.kt#213] [Q&A] [answered]
I need features that involve applying custom effects to the camera, similar to Snapchat's video effect capabilities. Can anyone suggest how I can implement functionality to process each frame using a deep learning algorithm and display the results as an overlay or modify the texture based on specific coordinates?

**Comment by shogo4405** [comment, marked as answer]
I provide simple filter functions such as changing the camera image to sepia or altering colors. You can verify this with example code. On the other hand, I'm not familiar with methods like utilizing a deep learning algorithm from another library. HaishinKit is an open-source software, so feel free to customize it if necessary.
